### PR TITLE
Use transparent theme for compose lambda

### PR DESCRIPTION
**Pull request by takahirom**
Fix https://github.com/takahirom/roborazzi/issues/67

### CODE REVIEW

**Comment by takahirom** [comment]
The failure of `unchanged check test / test (pull_request)` is OK. It is expected to fail.

**Review by yschimke — APPROVED** [review]
LGTM - although I wonder if it breaks some apps currently assuming a White background.  Coming from Wear, I think that's ok, as it wouldn't previously have a black background.

**Comment by yschimke** [comment]
It might be useful to demonstrate how to get back to a Theme background, should this be an option?

**Comment by takahirom** [comment]
I think the new Compose function hasn't been widely used yet. People can create their own custom captureRoboImage(). I'm going to mention this PR comment in the release notes.

```kotlin
fun captureRoboImageNoTransparent(
  file: File,
  roborazziOptions: RoborazziOptions = RoborazziOptions(),
  content: @Composable () -> Unit,
) {
  if (!roborazziEnabled()) return
  // Use non-transparent activity
  val activityScenario = ActivityScenario.launch(ComponentActivity::class.java)
  activityScenario.onActivity { activity ->
    activity.setContent {
      content()
    }
    val composeView = activity.window.decorView
      .findViewById<ViewGroup>(android.R.id.content)
      .getChildAt(0) as ComposeView
    val viewRootForTest = composeView.getChildAt(0) as ViewRootForTest
    viewRootForTest.view.captureRoboImage(file, roborazziOptions)
  }
}
```

File: roborazzi/src/main/java/com/github/takahirom/roborazzi/Roborazzi.kt [hunk 1]
```diff
@@ -187,7 +186,7 @@ fun captureRoboImage(
   content: @Composable () -> Unit,
 ) {
   if (!roborazziEnabled()) return
-  val activityScenario = ActivityScenario.launch(ComponentActivity::class.java)
+  val activityScenario = ActivityScenario.launch(RoborazziTransparentActivity::class.java)
```
Comment (takahirom) [inline roborazzi/src/main/java/com/github/takahirom/roborazzi/Roborazzi.kt:189]: image

**Comment by takahirom** [comment]
However, it seems difficult to view images without a background color. We are not currently using background colors in other parts, and users can use Rgb565 as a workaround. Therefore, I believe it is a good idea to use transparency in this PR.
image

**Comment by yschimke** [comment]
Transparent works for me, once this is allowed you can choose any other strategy.  But critically if transparent isn't supported, it's hard to put it back later.

I actually found the background was the big difference between Screen and Component level screenshots. Maybe something you could support different strategies, Fill, 50PctAlpha, Transparent, DeviceTheme?

For example

https://github.com/google/horologist/blob/main/roboscreenshots/src/main/java/com/google/android/horologist/screenshots/ScreenshotTestRule.kt#L248

```
    @Composable
    private fun ComponentDefaults(
        fakeImageLoader: FakeImageLoader,
        content: @Composable () -> Unit
    ) {
        fakeImageLoader.override {
            Box(
                modifier = Modifier
                    .wrapContentSize()
                    .background(Color.Black.copy(alpha = 0.5f)),
                contentAlignment = Alignment.Center
            ) {
                DefaultTheme {
                    content()
                }
            }
        }
    }
```

**Review by yschimke — APPROVED** [review]

**Pull request merged**